### ISSUE TITLE
Tell users not to use conditional reveals to reveal text (for example contextual help)

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-design-system#1362]
## What

Tell users not to use conditional reveals to reveal text (for example contextual help) – they should only be used to reveal related form fields. Any other use would need to be thoroughly research and tested.

This guidance should be added to the 'Conditionally revealing content' section for both the checkboxes and radios components.

## Why

As part of an audit of GOV.UK, the conditional reveals on https://www.gov.uk/email-signup?link=/world/switzerland were flagged as failing [WCAG 4.1.2](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html).

![Screenshot 2020-09-24 at 09 26 56](https://user-images.githubusercontent.com/121939/94120761-6abb9480-fe48-11ea-9ecd-db43c8050083.png)

The page uses the conditional reveals in an unusual way, as what’s being revealed is effectively contextual help (like a hint) that would help the user to make the right choice. Whereas the way we currently intend the component to be used (as per the examples on https://design-system.service.gov.uk/components/radios/#conditionally-revealing-content) is to reveal additional form controls that ‘follow on’ from the radios.

It fails because changes to the expanded ‘state’ of the radio buttons are not always available to assistive tech, and so users may be unaware that something else has changed on the page as a result of selecting a radio. This is true regardless of what type of content is being revealed.

However, when used to reveal form controls, we think that the user is very likely to discover the revealed control as they navigate forwards from the radio button or checkbox. When used to reveal text, the text is not focusable and so is unlikely to be discovered.

We therefore want to discourage users from using conditional reveals in this way, whilst we investigate solutions to the broader issue.

## Who needs to know about this
Content Designer

## Done when
- [x] Drafted update to guidance
- [ ] 2i'd 
- [ ] Published

**Comment by edwardhorsford** [comment]
I've used this pattern before successfully - though I don't recall specifically testing this with AT.

Would a similar issue occur if it revealed a form control that was prefaced with a heading / paragraph of text?

**Comment by joelanman** [comment]
yep, the user wouldnt know about the text, just the form control because they move to it

**Comment by 36degrees** [comment]
I think it'll depend on how they're navigating through the page. I don't think it's true to say they definitely wouldn't know about the text, but it's a lot more likely that they'd miss it.

**Comment by jbuller** [comment]
I think your are referring being in forms mode as opposed to browse mode of a screenreader.
In forms mode the elements that are not (programmatically associated with) fields may not are not read out.
